lib/fluent/plugin/out_detect_exceptions.rb

```diff
@@ -84,17 +86,19 @@ def emit(tag, es, chain)
 
     def process_record(tag, time_sec, record)
       synchronize do
-        unless @accumulators.key?(tag)
+        log_id = [tag]
+        log_id.push(record.fetch(stream, '')) unless stream.empty?
```

Comment:
[Optional] I think the config parameters manifest as instance fields, so it might pay to be explicit and use `@stream` instead of just `stream` (makes code searches much simpler too). Also for the other config parameters -- sorry I didn't pick up on this earlier. Feel free to defer this to a later cleanup.